docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,514 @@
+          value="up"
+        />
+        <RadioButton
+          checked={idealDirection === 'right'}
+          id="right"
+          label="Right"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'right' )}
+          value="right"
+        />
+        <RadioButton
+          checked={idealDirection === 'down'}
+          id="down"
+          label="Down"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'down' )}
+          value="down"
+        />
+        <RadioButton
+          checked={idealDirection === 'left'}
+          id="left"
+          label="Left"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'left' )}
+          value="left"
+        />
+      </Flex>
+      <Tooltip
+        idealDirection={idealDirection}
+        inline
+        text="Share"
+        >
+          <IconButton
+            accessibilityLabel="Share this Pin"
+            bgColor="white"
+            icon="share"
+            iconColor="darkGray"
+            inline
+            size="lg"
+          />
+        </Tooltip>
+      </Flex>
+
+  )
+}
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Link"
+      description="Passing in a link will render the link in the tooltip."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 <Tooltip
-  inline
   text="Logout"
   link={
-    <Link href="https://pinterest.com" target="blank">
-      <Text color="white" size="sm" weight="bold">
+    <Text color="white" size="sm" weight="bold">
+      <Link href="https://pinterest.com" target="blank">
         Learn more
-      </Text>
-    </Link>
+      </Link>
+    </Text>
   }
 >
-  <Icon
+  <IconButton
     accessibilityLabel="Logout of your profile"
-    color="pine"
-    icon="logout"
+    bgColor="white"
+    icon="share"
+    iconColor="darkGray"
+    size="lg"
   />
 </Tooltip>
 `}
-  />,
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Z-index"
+      description="Tooltip uses a Layer component to correctly position itself along the z-axis. The example below shows a FixedZIndex used in Modal and a CompositeZIndex to layer the Tooltips on top of it."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function ScrollableContainerExample() {
+  const [showModal, setShowModal] = React.useState(false);
+  const [alignText, setAlignText] = React.useState('left')
+  const MODAL_Z_INDEX = new FixedZIndex(11);
+  const TOOLTIP_Z_INDEX = new CompositeZIndex([MODAL_Z_INDEX]);
+
+  return (
+    <>
+      <Box
+        display="flex"
+        justifyContent="center"
+      >
+        <IconButton
+          accessibilityLabel="Edit this Pin"
+          bgColor="white"
+          icon="edit"
+          iconColor="darkGray"
+          inline
+          onClick={() => setShowModal(true)}
+          size="lg"
+          text="Edit Pin"
```

Comment:
Is this Text supposed to be on a Tooltip? I think a regular Button saying "Open edit modal" might be better here

Reply:
DONE